website/docs/guides/orchestration/webhooks/zapier-ms-teams.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "Post to Microsoft Teams when a dbt Cloud job finishes"
+id: webhooks-guide-zapier-ms-teams
+slug: zapier-ms-teams
+description: Use Zapier and webhooks to send info to Microsoft Teams
+---
+
+This guide assumes familiarity with:
+- dbt Cloud webhooks
+- Zapier
+
+## What does this integration do? 
+When a dbt Cloud job completes:
```

Comment:
```suggestion
When a dbt Cloud job finishes running, the integration will:

 - Receive a webhook notification in Zapier
 - Extract the results from the dbt Cloud admin API 
 - Post a summary to a Microsoft Teams channel
```

Reply:
I would suggest either listing this as numbers (because this is a sequential thing that happens), OR keeping the bullets and adding some `ands` and commas, so it would read like:

- Receive a webhook notification in Zapier,
- Extract the results from the dbt Cloud admin API,
- and post a summary to a Microsoft Teams channel.

So a reader knows all three of these things happen in a sequential order